packaging/functions.sh

```diff
@@ -244,6 +244,7 @@ set_mod_version() {
 # Copy launch wrappers, application icons, desktop, and MIME files to the target directory
 # Arguments:
 #   SRC_PATH: Path to the root OpenRA directory
+#   DEST_PATH: Prefix for the install path (Never put it into the executables!)
```

Comment:
This doesn't have any of the understandability from your explanation in the PR comments... how about
```
BUILD_PATH: Temporary path where the wrapper scripts are created (e.g. /tmp/openra-build/)
```

where you could replace my fake path with a real example from your packaging.

Reply:
Yes, that seems more descriptive. I hope that my advice is still not forgotten in the next refactor of the build system.